src/components/CommentList.jsx

```diff
@@ -1,23 +1,48 @@
+import { Fragment } from "react";
 import {
+  IconButton,
   Box,
   Divider,
   List,
   ListItem,
   ListItemText,
   Typography,
 } from "@mui/material";
+import DeleteIcon from "@mui/icons-material/Delete";
+import { useUser } from "../contexts/User";
+import { apiClient } from "../../api";
 
 const CommentList = (props) => {
+  const { loggedInUser, setLoggedInUser } = useUser();
+
+  function handleClick(id) {
+    apiClient.delete(`/api/comments/${id}`).then(() => {
+      // refresh the window upon deletion
+      window.location.reload();
```

Comment:
can we change this please, right now when a user deletes a comment the page is reloaded and they are logged out and most users would find this annoying. let aim to allow the user to delete the comment but but have them stay logged in after they delete the comment and also show them that the comment has been deleted